src/AzureAuth.Test/CommandMainTest.cs

```diff
@@ -23,6 +23,7 @@ namespace Microsoft.Authentication.AzureAuth.Test
     using NLog.Extensions.Logging;
     using NLog.Targets;
     using NUnit.Framework;
+    using static System.Formats.Asn1.AsnWriter;

```

Comment:
Did this get added by mistake or am I missing something? 🤔

Reply:
Accidently